src/handlebars/archive.handlebars

```diff
@@ -60,16 +70,16 @@
                           \{{else}}
                             <td></td>
                           \{{/if}}
-                          <td class='download-td'>
-                            <a class='blue-button no-underline' href='\{{link}}'><var binary-info>\{{type}} (\{{size}} MB)</var></a>
-                          </td>
                           \{{#if installer_link}}
                           <td class='download-td'>
-                            <a class='blue-button no-underline' href='\{{installer_link}}'><var binary-info>Installer \{{type}}</var></a>
+                            <a class='blue-button no-underline' href='\{{installer_link}}'><var binary-info>\{{type}} (\{{installer_extension}})</var></a>
                           </td>
                           \{{else}}
-                          <td></td>
+                          <td><div class='empty-download'>&nbsp;</div></td>
```

Comment:
Unusual to have a div inside a td like this.

Reply:
it's because I had to "fake" a button essentially.